packages/cli/test/unit/commands/certs/info.unit.test.ts

```diff
@@ -0,0 +1,146 @@
+import {stdout, stderr} from 'stdout-stderr'
+import Cmd from '../../../../src/commands/certs/info'
+import runCommand from '../../../helpers/runCommand'
+import expectOutput from '../../../helpers/utils/expectOutput'
+import heredoc from 'tsheredoc'
+import * as nock from 'nock'
+import {
+  Endpoint,
+  endpoint,
+  endpointWithDomains,
+  endpointUntrusted,
+  endpointTrusted,
+  certificateDetails,
+  certificateDetailsWithDomains,
+} from '../../../helpers/stubs/sni-endpoints'
+const sharedSni = require('./shared_sni.unit.test.ts')
+
+describe('heroku certs:info', function () {
+  it('shows certificate details when self-signed', async function () {
+    nock('https://api.heroku.com')
+      .get('/apps/example/sni-endpoints')
+      .reply(200, [endpoint])
+    nock('https://api.heroku.com', {
+      reqheaders: {Accept: 'application/vnd.heroku+json; version=3'},
+    })
+      .get('/apps/example/sni-endpoints/tokyo-1050')
+      .reply(200, endpoint)
+    await runCommand(Cmd, ['--app', 'example'])
+    expectOutput(stderr.output, heredoc(`
+      Fetching SSL certificate tokyo-1050 info for ⬢ example...
+      Fetching SSL certificate tokyo-1050 info for ⬢ example... done
+    `))
+    expectOutput(stdout.output, heredoc(`
+Certificate details:
+${certificateDetails()}
```

Comment:
Indent the terminal output when using `heredoc` to improve readability. Also, I believe to have added `tsheredoc` for the same purpose. I don't know if it makes sense to use one single package for that.

Reply:
so i was using heredoc, but for reasons that are unclear to me it was having problems with indents on the first line. it may have something to do with calling heredoc on something that was already heredoc'd.. or functions that returns strings that are then heredoc'd? i dunno, at any rate, thanks for pointing this out. i went ahead and reworked stuff so that heredoc will work on all these places.